examples/purchaseTesterTypescript/.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.6
```

Comment:
I guess this is determined by RN right? Asking just in case we could use a 3.x version.

Reply:
yeah, you can see it as part of the upgrade steps suggested by RN's Upgrade Helper in the link in the PR description